Codeinwp/optimole-wp
# Backwards compatibility of offload with WordPress core below v6.0

### All Submissions:

 

### Changes proposed in this Pull Request:
 
- Fixes filter offload compatibility with WP versions lower than 6.0.0;
- We state compatibility with 5.5 as the lowest version, but in the media_offload class, we used to call a filter with more params than it had in versions below 6.0.0;

Closes #640.

### How to test the changes in this Pull Request:

1. Install the plugin on a fresh instance using WordPress 5.5 to 6.0 exclusively;
2. Connect the plugin and enable offload;
3. Attempt to import in a gallery context anything from the DAM (that is not already on the website);
4. No errors should be thrown;

### Other information:
